app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -13,20 +13,20 @@
 
 <p>If you’re studying for a degree and interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
```

Comment:
@philspray the degree eligibility has changed as per the doc of requested changes

wording to be tweaked as needed


```suggestion
<p>If you’re studying for an undergraduate degree or a master's degree and are interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
```